.github/workflows/publish.yml

```
@@ -1,23 +1,32 @@
-name: Publish to NPM
+name: Publish
 on:
   release:
     types: [ created ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)